### DISCUSSION TITLE
축하드립니다. (디지털 정부서비스 UIUX 가이드라인 관련)

### DISCUSSION THREAD

**Discussion by kms0219kms** [orioncactus/pretendard#193] [Show and tell]
어제 배포된 디지털 정부서비스 UIUX 가이드라인에 서체로 Pretendard GOV가 강제되었네요. 축하드립니다. @orioncactus 

![IMG_7206](https://github.com/orioncactus/pretendard/assets/67222970/47ca8939-8917-4f7b-83af-42293cf610d2)

**Comment by orioncactus** [comment]
축하해주시고 알려주셔서 감사드립니다! 여러 환경에서 쾌적한 타이포그래피를 보여줄 수 있기를 기대합니다 💖

**Reply by kms0219kms** [reply]
저 가이드라인의 단점은 제정된 2024년 2월 이후 개발이 진행된 디지털정부서비스 선정 대상 웹/앱에만 적용된다는 겁니다. 

정부24같은 사이트에서도 아직은 반영을 기대하기 어렵다능 뜻인데요.

그래도 앞으로 쾌적한 타이포그래피를 보여줄 수 잇기를 저도 기대하겠습니다.
